client/src/components/MapView.vue

```diff
@@ -51,35 +52,58 @@ export default defineComponent({
     },
   },
   methods: {
+    /**
+     * Sets the visibility of the layer with the given styleLayerId.
+     */
+    setDataLayerVisibility(styleLayerId: string, visible: boolean): void {
+      if (this.map == null) return;
+
+      const visibility = visible ? 'visible' : 'none';
+      this.map.setLayoutProperty(styleLayerId, 'visibility', visibility);
```

Comment:
nit: If I understand typescript correctly, I think this method (and similar ones below) could be a cute one-liner:

```
this.map?.setLayoutProperty(styleLayerId, 'visibility', visible ? 'visible' : 'none');
```

(but concise !== readable, so feel free to ignore this)

Reply:
Done.